stl/inc/xutility

```diff
@@ -130,9 +130,13 @@ _NODISCARD constexpr void* _Voidify_iter(_Iter _It) noexcept {
 
 #if _HAS_CXX20
 template <class _Ty, class... _Types,
+#if 0 // TRANSITION, DevCom-XXXX, DevCom-XXXX

```

Comment:
Calling out the obvious "we need these `XXXX`s filled in before committing" so there will be a comment on the conversation to immediately clarify why this is blocked.

Reply:
Shouldn't this be `#ifdef __EDG__`

Reply:
Sure, because the workaround doesn't help EDG we can use original code for EDG.